docs/concepts/pools/boosted.md

```diff
@@ -6,73 +6,73 @@ order: 5
 
 ## Overview
 
-Boosted Pools are actually a subclass of [Composable Stable Pools](./composable-stable.md) but deserve their own page due to their powerful feature set. Boosted pools bring the best of both worlds to Liquidity Providers and Traders. Traders get access to deep stablecoin liquidity with near-parity exchange rates while Liquidity Providers get their liquidity positions sent to external protocols, such as [Aave](https://aave.com/).
+Boosted Pools are actually a subclass of other pools (typically [Composable Stable Pools](./composable-stable.md) and [Weighted Pools](./weighted.md)) but deserve their own page due to their powerful feature set. Boosted Pools bring the best of both worlds to Liquidity Providers (LPs) and Traders. Traders get access to deep liquidity with minimized price impact while Liquidity Providers get their liquidity positions sent to external protocols, such as [Aave](https://aave.com/).
 
-These versions of Stable Pools don't directly hold the stablecoins themselves, but rather hold the pool tokens (BPTs) of nested Linear Pools, which, for the Aave example, maintain proper balances of _TOKEN_ and _aTOKEN_.
+### Motivation
+In liquidity pools, swap prices are determined by pool balances. A large pool is great for traders: the larger the balances in the pool, the less the price changes from a given trade. The downside, however, is that large pools are not as great for LPs.
```

Comment:
The language around large pools being bad for LPs and diluting swap fees is slightly confusing because it makes it seem like it is worse than proportional - which imo would be someone's default assumption. May make sense to keep the focus on "large amounts of idle liquidity" being the negative

Reply:
gone